pages/common/borg.md

```diff
@@ -0,0 +1,32 @@
+# borg
+
+> Deduplicating backup tool with compression and encryption.
+> Supports local and remote backups, mountable as filesystems.
+
+- Initialise a (local) repository:
+
+`borg init {{/path/to/repo/folder}}`
+
+- Backup a folder into the repository, creating an archive called Monday:
+
+`borg create --progress {{/path/to/repo/folder::Monday}} {{/path/to/source/folder}}`
+
+- List all archives in a repository:
+
+`borg list {{/path/to/repo/folder}}`
+
+- Extract a specific folder from the 'Monday' archive, excluding all *.ext files:
+
+`borg extract {{/path/to/repo/folder::Monday}} {{path/to/target/folder}} --exclude '*.ext'`
```

Comment:
Perhaps you could put `{{` and `}}` around `*.ext`? Also, is the `'` meant to be there?

Reply:
The `'` are required, as far as I know. Yes, might be more consistent to add `{{` and `}}`.

Reply:
Ah, I see. Thanks!